data/human/boarding missions.txt

```diff
@@ -59,9 +61,82 @@ mission "Assisting Merchant (Large)"
 		not attributes "automaton"
 	on offer
 		payment 30000
-		conversation "assisting merchant"
+		conversation
+			`${assist merchant 1} <origin>${assist merchant 2} ${assist merchant 3} <payment>${assist merchant 4}`
 	destination "Earth"
 
+phrase "assist merchant 1"
+	word
+		`Knowing that `
+		`Since `
+		`Understanding that `
+		`Considering that `
+		`Seeing how `
+		`Knowing how `
+	word
+		`you risked your life `
+		`you risked your ship `
+		`you risked your own ship `
+		`you braved enemy fire `
+		`you braved danger `
```

Comment:
It might not always be the case that you risked yourself to repair a ship. It could easily be the case that there were no hostiles in the system when you repaired the ship, or even that you were friendly with whatever faction disabled the merchant ship. The existing conversation for the human merchant missions only states that the captain is thankful for your help. It's only the Hai boarding mission that talks about danger. Perhaps it's not out of hand for the Hai mission since the only place you'd realistically be repairing ships is in Wah Ki, where there's almost always danger, but I don't think it fits the best for the normal human merchant conversations.

Perhaps the Hai boarding mission and human merchant boarding missions get different phrases.